x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.tsx

```diff
@@ -153,201 +148,196 @@ export const EditPolicy: React.FunctionComponent<Props> = ({ history }) => {
   };
 
   return (
-    <EuiPage>
-      <EuiPageBody>
-        <EuiPageContent>
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle size="l" data-test-subj="policyTitle">
-                <h1>
-                  {isNewPolicy
-                    ? i18n.translate('xpack.indexLifecycleMgmt.editPolicy.createPolicyMessage', {
-                        defaultMessage: 'Create policy',
-                      })
-                    : i18n.translate('xpack.indexLifecycleMgmt.editPolicy.editPolicyMessage', {
-                        defaultMessage: 'Edit policy {originalPolicyName}',
-                        values: { originalPolicyName },
-                      })}
-                </h1>
-              </EuiTitle>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiButtonEmpty
-                href={createDocLink('index-lifecycle-management.html')}
-                target="_blank"
-                iconType="help"
-              >
+    <ManagementPageLayout
+      pageHeader={{
+        pageTitle: (
+          <span data-test-subj="policyTitle">
```

Comment:
I guess there's not a way to pass a `data-test-subj` to the page title?

Reply:
No it's not supported. Per https://github.com/elastic/kibana/pull/100085#discussion_r639371240 there is a hard-coded `data-test-subj` for the title created by `ManagementPageLayout` so we can migrate our code and tests over to use that instead I think. But it's also not a blocker IMO so I've decided not to implement that as part of this migration.